CHANGES/2784.bugfix

```diff
@@ -0,0 +1 @@
+Fix websocket client return EofStream.
```

Comment:
Is this file here by accident?

Reply:
It is previous PR, maybe i should take a new branch